packages/ui/src/models/visualization/flows/camel-rest-visual-entity.ts

```diff
@@ -66,31 +75,21 @@ export class CamelRestVisualEntity extends AbstractCamelVisualEntity<{ rest: Res
     const schema = CamelCatalogService.getComponent(CatalogKind.Entity, 'rest');
 
     return {
-      definition: Object.assign({}, this.rest),
+      definition: Object.assign({}, this.restDef.rest),
       schema: schema?.propertiesSchema || {},
     };
   }
 
   getOmitFormFields(): string[] {
-    return SchemaService.OMIT_FORM_FIELDS;
-  }
-
-  updateModel(path: string | undefined, value: unknown): void {
-    if (!path) return;
-
-    setValue(this.rest, path, value);
-
-    if (!isDefined(this.rest)) {
-      this.rest = {rest: Rest};
-    }
+    return this.OMIT_FORM_FIELDS;
   }
 
   getNodeInteraction(): NodeInteraction {
     return {
       canHavePreviousStep: false,
       canHaveNextStep: false,
       canHaveChildren: false,
-      canHaveSpecialChildren: true,
+      canHaveSpecialChildren: false,

```

Comment:
Since we are not providing the appropriate `get`, `post`, etc... nodes, would it be better to don't show this interaction?